### ISSUE TITLE
Sorry i'm dumb but please help me.

### ISSUE THREAD

**Issue by crimex2997** [hardikvasa/google-images-download#328]
How to use this.
I have zero knowledge in python if you round it down.
I tried googling for this but i still can't find the answer. But i think i installed everything.

Please, if someone will answer me, dumb it down as much as possible.

![image](https://user-images.githubusercontent.com/26241901/94138436-b860ed00-fe9a-11ea-9720-026d71081cf8.png)

**Comment by Joeclinton1** [comment]
You need to use the command "python setup.py install". You could also just double click on "setup.py" in file explorer.

If you are wondering why you need to use install, it's because install is one of the modules in setup.py and so to run it you put install after setup.py.

**Comment by crimex2997** [comment]
> You need to use the command "python setup.py install". You could also just double click on "setup.py" in file explorer.

if i double click the setup.py, it just open my pycharm.

also this is what happened after i run the py through cmd in the folder 


![image](https://user-images.githubusercontent.com/26241901/94140353-b2204000-fe9d-11ea-941d-3e9e16ff8ec5.png)

**Comment by crimex2997** [comment]
![Uploading image.png…]()

**Comment by crimex2997** [comment]
![image](https://user-images.githubusercontent.com/26241901/94170213-5ae09680-fec2-11ea-9bde-16984344ecf4.png)

**Comment by Joeclinton1** [comment]
I think it installed correctly.

**Comment by crimex2997** [comment]
so anyone know what's wrong?

**Comment by kevinkit** [comment]
I am not sure but I think you switched some things here, but installing it won't download anything. You need to execute some command like: `googleimagesdownload --keywords "universe" --usage_rights labeled-for-reuse` . However the repo is currently broken, sadly, see #325

**Comment by shaniafrincella** [comment]
Hi,

I am a Computer Science student from Coventry University. I am interested to contribute to this open source project. My area of contribution will be adding an option to open the search image in the web browser, adding a GUI to choose the file directory to save images and adding an image viewer GUI.

**Comment by hardikvasa** [comment]
Please check the documentation at https://google-images-download.readthedocs.io for usage instructions. For command-line usage, try: googleimagesdownload --keywords 'cats' --limit 10. For Python usage, see the examples in the README.
